src/App/Components/PasswordReset/ForgotPassword.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axiosInstance from '../../../configs/axios';
+import Alert from '../Common/Alert';
+
+export class PasswordReset extends Component {
+  state = { email: '' };
+
+  onChange = event => this.setState({ [event.target.name]: event.target.value });
+
+  onSubmit = async (event) => {
+    const { state } = this;
+    event.preventDefault();
+    await axiosInstance
+      .post('/api/password-reset', { email: state.email })
```

Comment:
@nkpremices this API call is not inside a redux action file. I'm curious about why.

Reply:
This component has nothing to do with other components. Its state is separate and unique. In the logic, there is not a place where I share this state with another component. That's the main reason why I didn't use redux @MemunaHaruna

Reply:
@nkpremices Using redux helps with more than just state sharing across components.
- It helps keep components clean
- knowing that all API stuff happens in the actions, this makes it easier for debugging or to make changes to the app
- it helps make the app code consistent across components
- it makes testing easier

Reply:
Oh! Now I see.
I wasn't aware of that. 
All I knew is just about sharing the state.
According to what you've just said, I see that It would have been better to implement redux.
let me do it @MemunaHaruna